R/package.R

```diff
@@ -35,6 +35,7 @@ send_headers <- c("Accept" = "application/vnd.github.v3+json",
 #' @param endpoint GitHub API endpoint. See examples below.
 #' @param ... Additional parameters
 #' @param .token Authentication token.
+#' @param .api_url Github API url (default: \url{https://api.github.com})
```

Comment:
No, the README is not necessarily comprehensive. They should both be here. But I can add them together, no worries.